servers/rendering_server.h

```diff
@@ -77,6 +85,22 @@ class RenderingServer : public Object {
 	RID white_texture;
 	RID test_material;
 
+	uint64_t variance_total_time = 0ul;
+	uint64_t avg_total_time = 0ul;
+	// Calling screen_get_refresh_rate() is expensive enough to appear in profilers.
+	// Plus it can cause internal mutexes in the OS that cause stutter. So it's cached.
+	uint64_t cached_refresh_rate_us = (1000ul * 1000ul) / 60ul; // Refresh rate in microseconds.
+	uint32_t cached_refresh_rate_mHz = 1000u * 60u; // Refresh rate in millihertz (mHz).
```

Comment:
Use snake_case for variable names:

```suggestion
	uint32_t cached_refresh_rate_mhz = 1000u * 60u; // Refresh rate in millihertz (mHz).
```

Reply:
I knew this one would be problematic.

The problem is that "mhz" can mean either **milli**hertz or **mega**hertz. Since "mhz" often is associated with megahertz; I used mHz on purpose. Suggestions are welcome.